### PR TITLE
feat: google cloud http fuction for telegram

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

**Comment by timyates** [comment]
Google cloud functions needs java 11 https://ge.micronaut.io/s/ntk5qm2bxrjam/console-log?task=:chatbots-telegram-google-cloud-http-function:compileJava

File: chatbots-telegram-google-cloud-http-function/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        substitute(module("io.micronaut:micronaut-jackson-databind"))
+            .using(module("io.micronaut.serde:micronaut-serde-jackson:${mn.versions.micronaut.serialization.get()}"))
+    }
+}
+
+
```
Comment (timyates) [inline chatbots-telegram-google-cloud-http-function/build.gradle.kts:19]: Do we need to add

```suggestion
micronautBuild {
    sourceCompatibility = "11"
    targetCompatibility = "11"
}
```

In here?

Does that make this whole module Java 11+ only?  Or just this library?

**Review by timyates — APPROVED** [review]

File: chatbots-telegram-google-cloud-http-function/build.gradle.kts (hunk 1 above)
Comment (sdelamo) [reply chatbots-telegram-google-cloud-http-function/build.gradle.kts]: not sure

Comment (sdelamo) [reply chatbots-telegram-google-cloud-http-function/build.gradle.kts]: ping @melix

**Pull request merged**